### ISSUE TITLE
[Death Knight] Threat of Thassarian; Death Strike conjunction

### ISSUE THREAD

**Issue by Peachusftw** [FeenixServerProject/Wrath_3.3.5_Issue_tracker#213]
Hey there,
The Threat of Thassarian talent (http://wotlk.openwow.com/spell=66192) does not cause Death Strike (http://wotlk.openwow.com/spell=49924) to heal twice for every disease existing on target. While the tooltip only mentions the damage part, the healing has been covered in countless topics and mostly deemed as intented, and thus, implemented. (example: http://eternal-wow.com/forum/confirmed-f356/death-strike-threat-of-thassarian-hit-1x-diseases-t98426.html).

Basically what is happening: If a ToT specced DK uses Death Strike, it heals for 5% oh his max HP per disease on target.
What should happen: Death Strike should heal for 10% (2x5%) of max HP per disease if DK has ToT talent.
Unfortunately both Wowwiki and Wowhead pages have been updated and would no longer be of any use.

**Comment by dangreenberggit** [comment]
I'm gonna change your life
http://wowwiki.wikia.com/wiki/Death_Strike?oldid=2412594
http://wowwiki.wikia.com/wiki/Threat_of_Thassarian?oldid=2324843
